gum/arch-arm/gumthumbwriter.c

```diff
@@ -788,6 +788,50 @@ gum_thumb_writer_put_ldr_reg_reg_offset (GumThumbWriter * self,
       GUM_THUMB_MEMORY_LOAD, dst_reg, src_reg, src_offset);
 }
 
+gboolean
+gum_thumb_writer_put_vldr_reg_reg_offset (GumThumbWriter * self,
+                                         arm_reg dst_reg,
+                                         arm_reg src_reg,
+                                         gsize src_offset)
+{
+  GumArmRegInfo src_reg_info;
+  GumArmRegInfo dst_reg_info;
+
+  // cccc cccc cycc ssss dddd cccx oooo oooo
+  // c: opt code fixed
+  // y: dest reg index, bit 1 
+  // s: source reg index
+  // d: dest reg index, bit 2-5
+  // x: 0 f32 load  1 f64 load
+
+  uint32_t code = 0xed900a00;
+  gboolean is_float = TRUE;
```

Comment:
These two variables should be moved up right after the other two variable declarations.